spec/views/content_items/index.html.erb_spec.rb

```diff
@@ -50,109 +51,77 @@
         render
 
-        expect(rendered).to have_text('An organisation')
+        expect(rendered).to have_text('organisation links')
       end
-    end
-  end
 
-  context "filter by org" do
-    let(:organisation) { build(:organisation) }
-    let(:content_items) { ContentItemsDecorator.new(build_list(:content_item, 2)) }
+      it "renders item document type" do
+        allow(content_items[0]).to receive(:document_type).and_return("a document type")
 
-    before do
-      allow(view).to receive(:paginate)
-      allow(view).to receive(:page_entries_info)
-      assign(:content_items, content_items)
-      assign(:organisation, organisation)
-      assign(:organisations, [])
-      assign(:taxonomies, [])
-    end
-
-    it 'renders the title of the organisation' do
-      allow(content_items).to receive(:header).and_return('A Title')
-      render
-
-      expect(rendered).to have_selector('h1', text: 'A Title')
-    end
-
-    it 'renders the table header with the right headings' do
-      render
-
-      expect(rendered).to have_selector('table thead', text: 'Title')
-      expect(rendered).to have_selector('table thead tr:first-child th:nth(2)', text: 'Type of Document')
-      expect(rendered).to have_selector('table thead tr:first-child th:nth(3)', text: 'Unique page views (last 1 month)')
-      expect(rendered).to have_selector('table thead tr:first-child th:nth(4)', text: 'Last Updated')
-    end
-
-    it 'renders a row per Content Item' do
-      render
-
-      expect(rendered).to have_link(content_items.first.title, href: content_item_path(id: content_items.first.id))
-    end
-
-    it 'returns the max page size number of content items per page' do
-      render
-      expect(rendered).to have_selector('table tbody tr', count: 2)
-    end
-
-    describe 'Kaminari' do
-      it 'uses Kaminari to paginate the pages' do
-        expect(view).to receive(:paginate)
         render
-      end
 
-      it 'returns the max page size number of content items per page' do
-        render
-        expect(rendered).to have_selector("table tbody tr", count: 2)
+        expect(rendered).to have_text('a document type')
       end
-    end
 
-    describe 'row content' do
-      it 'includes the content item title' do
-        content_items[0].title = 'a-title'
+      it "renders item page views" do
+        allow(content_items[0]).to receive(:unique_page_views).and_return("1234")
+
         render
 
-        expect(rendered).to have_link('a-title', href: content_item_path(id: content_items.first.id))
+        expect(rendered).to have_text('1234')
       end
 
-      it 'includes the last time the content was updated' do
+      it "renders item last updated" do
         Timecop.freeze(Time.parse('2016-3-20')) do
-          content_items[0].public_updated_at = Time.parse('2016-1-20')
+          allow(content_items[0]).to receive(:public_updated_at).and_return(Time.parse('2016-1-20'))
```

Comment:
This is done partly for consistency with the other assertions in this file - but also for robustness if this value switches to a decorator or similar - the view should only ensure it rendered whatever is returned from that `:message` being sent.